6238_prototype/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -70,16 +71,12 @@ private void addBall() {
                 .whenReleased(new RetractIntakeCommand(ballSubsystem));
 
         new JoystickButton(joystick, IOConstants.START_SHOOTER)
-                .whenPressed(new ShooterCommand(ballSubsystem))
-                .whenReleased(new StopShooterCommand(ballSubsystem));
-
-        new JoystickButton(joystick, IOConstants.RIGHT_KICKER)
-                .whenPressed(() -> BallSubsystem.activateRightKicker(true))
-                .whenReleased(() -> BallSubsystem.activateRightKicker(false));
-
-        new JoystickButton(joystick, IOConstants.LEFT_KICKER)
-                .whenPressed(() -> BallSubsystem.activateLeftKicker(true))
-                .whenReleased(() -> BallSubsystem.activateLeftKicker(false));
+                .whenPressed(
+                        new SequentialCommandGroup(
+                                new ShooterCommand(ballSubsystem, PneumaticKickers.LEFT_KICKER, 4000),
+                                new ShooterCommand(ballSubsystem, PneumaticKickers.RIGHT_KICKER, 4000)
+                        )
+                );
```

Comment:
We need to have a StopShooter command at the end of the command group and also `whenReleased`.  We want the driver to be able to stop the shooter by releasing the trigger.